dotCMS/src/main/java/com/dotcms/rest/api/v1/categories/CategoriesResource.java

```diff
@@ -244,11 +249,52 @@ public final Response getChildren(@Context final HttpServletRequest httpRequest,
         DotPreconditions.checkArgument(UtilMethods.isSet(inode),
                 "The inode is required");
 
+        return getChildrenCategories(user, inode, pageMode.respectAnonPerms, page, perPage,
```

Comment:
we already have a pattern for pagination, if you need to return a diff type, instead of using the pagination as a feed, create a new paginator that returns the type and the info you need

Reply:
This feedback has been addressed. @jdotcms can you please review it